search/src/main/kotlin/com/ritense/search/domain/SearchListColumn.kt

```diff
@@ -40,7 +40,7 @@ data class SearchListColumn(
     @Column(name = "path")
     val path: String,
     @Column(name = "column_order")
-    val order: Int,
+    var order: Int,
```

Comment:
Instead of making this a `var`, you could use `.copy(..)` as it's a data class

Reply:
fixed